### PR TITLE
Use the latest default version of Rocoto on RDHPCS platforms

### PR DESCRIPTION
**Pull request by BijuThomas-NOAA**
**Description**

It was reccomended by Jet Admin to use the latest default version of Rocoto, rather than a specific version, as certain versions failed to track the status of the workflow with the Slurm upgrade on RDHPCS platforms(Issue https://github.com/hafs-community/HAFS/issues/254)

The following changes are included in this PR.

- The application level modulefiles for Hera/Orion/Jet were modified to use the default version of Rocoto

## Application-level regression test status
Running the HAFS application-level regression tests is currently performed by code reviewers after the developer creates the initial PR. As regression tests are conducted, the testers should use the checklist below to indicate **successful** regression tests. You may add other tests as needed. If a test fails, do not check the box. Instead, describe the failure in the PR comments, noting the platform where the test failed.

- [x] Jet
- [x] Hera
- [x] Orion

### CODE REVIEW

**Comment by mrinalbiswas** [comment]
@BinLiu-NOAA RT on Hera passed successfully. RT on Jet still on queue. Is anyone running on Orion? I saw 3 platform in the PR.

**Comment by BijuThomas-NOAA** [comment]
> @BinLiu-NOAA RT on Hera passed successfully. RT on Jet still on queue. Is anyone running on Orion? I saw 3 platform in the PR.

@mrinalbiswas I am running RT on Orion

**Comment by BinLiu-NOAA** [comment]
Thanks, @BijuThomas-NOAA and @mrinalbiswas!

File: modulefiles/hafs.hera.lua
Comment (BinLiu-NOAA) [inline modulefiles/hafs.hera.lua:124]: @BijuThomas-NOAA, with the change to use `load("rocoto")`, do we still need to keep the line above `rocoto_ver=os.getenv("rocoto_ver") or "1.3.3"`? Or alternatively, will `rocoto_ver=os.getenv("rocoto_ver") or "default"` work? Thanks!

Comment (BijuThomas-NOAA) [reply modulefiles/hafs.hera.lua:124]: @BinLiu-NOAA:
We do not need to keep the line `rocoto_ver=os.getenv("rocoto_ver") or "1.3.3"`. 
I do not think `rocoto_ver=os.getenv("rocoto_ver") or "default"` will work

**Comment by mrinalbiswas** [comment]
@BinLiu-NOAA RT on Jet was successful.

File: modulefiles/hafs.hera.lua
Comment (BijuThomas-NOAA) [reply modulefiles/hafs.hera.lua:124]: RT passed on Orion

Comment (BijuThomas-NOAA) [reply modulefiles/hafs.hera.lua:124]: > @BijuThomas-NOAA, with the change to use `load("rocoto")`, do we still need to keep the line above `rocoto_ver=os.getenv("rocoto_ver") or "1.3.3"`? Or alternatively, will `rocoto_ver=os.getenv("rocoto_ver") or "default"` work? Thanks!

We do not keep the line `rocoto_ver=os.getenv("rocoto_ver") or "1.3.3"`
I do not think `rocoto_ver=os.getenv("rocoto_ver") or "default"` will work

**Review by BinLiu-NOAA — APPROVED** [review]
Thanks for the fix and testing!

**Comment by BinLiu-NOAA** [comment]
I think the PR is ready to merge once we get enough review approvals. Thanks!

**Review by KathrynNewman — APPROVED** [review]

**Pull request merged**
